### PR TITLE
phpdismod: add more information link

### PR DESCRIPTION
**Pull request by mere-teresa**
See #6062

- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/phpdismod.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # phpdismod
 
 > Disable PHP extensions on Debian-based OSes.
+> More information: https://linuxcommandlibrary.com/man/phpdismod
```
Comment (navarroaxel) [inline pages/linux/phpdismod.md:4]:
```suggestion
> More information: <https://salsa.debian.org/php-team/php-defaults>.
```

Comment (mere-teresa) [reply pages/linux/phpdismod.md:4]: Thank you.
I have a question about the website: which website is legit? How to be sure?

Comment (mere-teresa) [reply pages/linux/phpdismod.md:4]: Additional question, `phpdismod` is not so easy to find on the salsa.debian.org website, is it okay?

Comment (navarroaxel) [reply pages/linux/phpdismod.md:4]: I reviewed the source code that Debian use to build the package that contains this command, but if there is no man page available, I think the source code is the best that we can provide. `linuxcommandlibrary` doesn't add more information neither. :man_shrugging:

Comment (mere-teresa) [reply pages/linux/phpdismod.md:4]: 🆗  Thank you!

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]

**Pull request merged**